quartz/i18n/locales/nl-NL.ts

```diff
@@ -82,5 +82,16 @@ export default {
         count === 1 ? "Eerste label tonen." : `Eerste ${count} labels tonen.`,
       totalTags: ({ count }) => `${count} labels gevonden.`,
     },
+    encryptedContent: {
+      loading: "Bezig met laden...",
+      password: "Wachtwoord",
+      submit: "Verzenden",
+      enterPassword:
+        "Deze pagina is standaard vergrendeld. Voer het wachtwoord in om te ontgrendelen:",
+      modernBrowser: "Gebruik een moderne browser.",
+      wrongPassword: "Nieprawidłowe hasło. Wprowadź ponownie hasło, aby odblokować:",
```

Comment:
This is not Dutch.

Reply:
I used AI to translate. I don't know why AI translate these string to Polish :) I will recheck all translated string.